pkg/cmd/roachtest/tests/awsdms.go

```diff
@@ -203,14 +224,15 @@ func runAWSDMS(ctx context.Context, t test.Test, c cluster.Cluster) {
 		}
 	}()
 
-	sourcePGConn, err := setupAWSDMS(ctx, t, c, rdsCli, dmsCli)
+	sourcePGConn, err := setupAWSDMS(ctx, t, c, rdsCli, dmsCli, smCli)
 	if err != nil {
 		t.Fatal(err)
 	}
 	targetPGConn := c.Conn(ctx, t.L(), 1)
 
 	checkDMSReplicated(ctx, t, sourcePGConn, targetPGConn)
```

Comment:
can we use `t.Run` on all of these so that they come out as subtests?

Reply:
There doesnt appear to be a t.Run method on the t object

Reply:
forgot to delete this comment